testdata/project-v3-addon/README.md

```diff
@@ -0,0 +1,77 @@
+# project-v3-addon
+// TODO(user): Add simple overview of use/purpose
+
+## Description
+// TODO(user): An in-depth paragraph about your project and overview of use
+
+## Getting Started
+You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
+**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
+
+### Running on the cluster
+1. Install Instances of Custom Resources:
+	```sh
+	kubectl apply -f config/samples/
+	```	
+2. Build and push your image to the location specified by `IMG`:
+	```sh
+	make docker-build docker-push IMG=<some-registry>/<project-name>:tag
+	```	
+3. Deploy the controller to the cluster with the image specified by `IMG`:
+	```sh
+	make deploy IMG=<some-registry>/project-name>:tag
+	```	 
+### Uninstall CRDs
+To delete the CRDs from the cluster:
+	```sh
+	make uninstall
+	```	
+### Undeploy controller
+UnDeploy the controller to the cluster:
+	```sh
+	make undeploy
+	```	
+
+## Contributing
+// TODO(user): Add detailed information on how you would like others to contribute to this project
+
+### How it works
+This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+
+It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) 
+which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster 
+
+### Test It Out
+You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
+**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
```

Comment:
We already added it above. Maybe here is better we just clarifies that the following commends will help the author test it outside of the cluster

Reply:
Make we could put here a Markdown table with the main/most important targets or some tips like 
If you change the manifests ensure that you run make manifests etc. WDYT?
Ideas? Suggestions?

Reply:
> We already added it above. Maybe here is better we just clarifies that the following commends will help the author test it outside of the cluster

Oops, mistake from copying the code and moving it down that I missed - thanks!

> Make we could put here a Markdown table with the main/most important targets or some tips like If you change the manifests ensure that you run make manifests etc. WDYT? Ideas? Suggestions?

I like the idea of including the `make manifests` if you make changes to the API definitions since that is something that is included in the Kubebuilder quickstart so I will add that. I don't know if we should put a table of `make` targets because I think once we add the `make manifests` one that covers all the ones in the quickstart, which I would think are the most important ones to be aware of. However, since I am new to all of this if you think that there are some targets not included that should be I am happy to create a table of all the `make` targets already in the template and add any that you feel are missing.

Reply:
Changes implemented in https://github.com/kubernetes-sigs/kubebuilder/pull/2610/commits/a0faa6f08895abaf1bad432af586e49ef8ce107e

Let me know what you think